vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -87,7 +110,8 @@ if [ "${OS_NAME,,}" == "linux" ]; then
     "offer_name": "$OFFER_NAME",
     "hyperv_generation": "${HYPERV_GENERATION}",
     "image_architecture": "${IMAGE_ARCH}",
-    "image_version": "${IMAGE_VERSION}"
+    "image_version": "${IMAGE_VERSION}",
+    "replication_inverse": "${REPLICATION_INVERSE}"
```

Comment:
would it be possible to compute this at release-time instead?

Reply:
it is possible, yes. I dont have a strong preference of doing it here or there, I just thought its simpler to have the calc done here and just use the inverse from publishing info in RP.

But it is doable in RP as well, sure.

Reply:
one reason I did do it here is testability. its easier to build VHDs and see the replication inverse calculated here instead of behind ev2 logs :)

Reply:
this is true

Reply:
doing it here is fine, lets just make sure we have good reasons we've all agreed upon when adding stuff to publishing info moving forward